client/my-sites/migrate/step-upgrade.jsx

```diff
@@ -175,10 +175,10 @@ const WrappedStepUpgrade = ( props ) => {
 			{ ...props }
 			isEcommerceTrial={ isEcommerceTrial }
 			billingTimeFrame={ plan.getBillingTimeFrame() }
-			currencyCode={ pricingMeta[ planSlug ]?.currencyCode }
+			currencyCode={ pricingMeta?.[ planSlug ]?.currencyCode }
 			planPrice={
-				pricingMeta[ planSlug ]?.discountedPrice?.monthly ??
-				pricingMeta[ planSlug ]?.originalPrice?.monthly
+				pricingMeta?.[ planSlug ]?.discountedPrice?.monthly ??
+				pricingMeta?.[ planSlug ]?.originalPrice?.monthly
```

Comment:
```suggestion
				pricingMeta?.[ planSlug ]?.discountedPrice.monthly ??
				pricingMeta?.[ planSlug ]?.originalPrice.monthly
```

NIT - and no need to change anything - `discountedPrice` and `originalPrice` are always defined on compiled pricing